packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
     dest = temp[temp.length - 1];
   }
 
-  return dest;
+  if (isUUID.anyNonNil(dest)) {
+  // assume it's a convo ID
```

Comment:
I would replace this with a more comprehensive header at the top of the `if` statement. 

```
// If the destination is a UUID, assume it is an internal conversation ID.
// Next, assume it is a valid Hydra room ID. 
// If decoding fails, it is a SIP or other destination.
```

Reply:
done